packages/ncids-css/scss/components/nci-feature-card/_feature-card.scss

```diff
@@ -0,0 +1,71 @@
+/* Card (For Shadow) */
+.feature-card {
```

Comment:
should be  .nci-feature-card (follow [the way](https://github.com/NCIOCPL/ncids/wiki/The-Way) here)